src/integrationTest/java/uk/gov/hmcts/darts/audio/component/impl/AudioRequestBeingProcessedFromArchiveQueryImplIntTest.java

```diff
@@ -7,6 +7,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.darts.audio.component.AudioBeingProcessedFromArchiveQuery;
```

Comment:
I think I'd prefer to see a separate integration test for your query - however that looks?
`AudioBeingProcessedFromArchiveQueryImplIntTest`?
Do you agree @hemantasharma1129 ?

Reply:
have split into separate integration test - to prevent any arguments :-D